test/recipes/03-test_fipsinstall.t

```diff
@@ -27,75 +27,76 @@ plan skip_all => "Test only supported in a fips build" if disabled("fips");
 plan tests => 12;
 
 my $infile = bldtop_file('providers', platform->dso('fips'));
+my $fipskey = $ENV{FIPSKEY} // '00';
```

Comment:
_(Shouldn't this default to the holy handgrenade, too?)_

Reply:
I could go either way.  As it is right now, it catches a "make" error where the env var isn't set.

Reply:
I have no preference, so just leave it for the moment. Maybe @slontis has an opinion about it when he reapproves.

Reply:
As long as it runs I dont really mind..